custom_components/tuya_local/devices/klarstein_skytower_grand_smart.yaml

```diff
@@ -0,0 +1,79 @@
+name: Air cooler
+products:
+  - id: rlphjc5nzs3v5xdt
+    manufacturer: Klarstein
+    model: Skytower Grand Smart
+entities:
+  - entity: fan
+    translation_only_key: fan_with_presets
+    dps:
+      - id: 1
+        name: switch
+        type: boolean
+      - id: 2
+        type: string
+        name: speed
+        mapping:
+          - dps_val: "low"
+            value: 33
+          - dps_val: "mid"
+            value: 66
+          - dps_val: "high"
+            value: 100
+      - id: 3
+        type: string
+        name: preset_mode
+        mapping:
+          - dps_val: normal
+            value: normal
+          - dps_val: nature
+            value: nature
+          - dps_val: sleep
+            value: sleep
+      - id: 10
+        type: boolean
+        name: oscillate
+  - entity: select
+    translation_key: timer
+    category: config
+    dps:
+      - id: 11
+        type: string
+        name: option
+        mapping:
+          - dps_val: close
+            value: close
+          - dps_val: "1H"
+            value: "1h"
+          - dps_val: "2H"
+            value: "2h"
+          - dps_val: "3H"
+            value: "3h"
+          - dps_val: "4H"
+            value: "4h"
+          - dps_val: "5H"
+            value: "5h"
+          - dps_val: "6H"
+            value: "6h"
+          - dps_val: "7H"
+            value: "7h"
+          - dps_val: "8H"
+            value: "8h"
+          - dps_val: "9H"
+            value: "9h"
+  # - entity: sensor
```

Comment:
Normally a select style timer will come with such a sensor, so I think it should be included. Often if the timer is not set, then the dp is not reported, so the dp should be marked as `optional: true`